### ISSUE TITLE
Error when trying to compile as dependency

### ISSUE THREAD

**Issue by rcalxrc08** [sunoru/RandomNumbers.jl#47]
```
Julia Version 1.0.0
Commit 5d4eaca0c9 (2018-08-08 20:58 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-4500U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, haswell)
```

I get the following:
```
┌ Error: Error building `RandomNumbers`:
│ ERROR: LoadError: LoadError: UndefVarError: info not defined
│ Stacktrace:
│  [1] build() at C:\Users\nicol\.julia\packages\RandomNumbers\cpJea\deps\Random123\build.jl:13
│  [2] top-level scope at C:\Users\nicol\.julia\packages\RandomNumbers\cpJea\deps\Random123\build.jl:40
│  [3] include at .\boot.jl:317 [inlined]
│  [4] include_relative(::Module, ::String) at .\loading.jl:1038
│  [5] include(::Module, ::String) at .\sysimg.jl:29
│  [6] include(::String) at .\client.jl:388
│  [7] top-level scope at none:0
│  [8] include at .\boot.jl:317 [inlined]
│  [9] include_relative(::Module, ::String) at .\loading.jl:1038
│  [10] include(::Module, ::String) at .\sysimg.jl:29
│  [11] include(::String) at .\client.jl:388
│  [12] top-level scope at none:0
│ in expression starting at C:\Users\nicol\.julia\packages\RandomNumbers\cpJea\deps\Random123\build.jl:39
│ in expression starting at C:\Users\nicol\.julia\packages\RandomNumbers\cpJea\deps\build.jl:1
└ @ Pkg.Operations C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.0\Pkg\src\Operations.jl:1068
```

I think that `info` should be `@info` in https://github.com/sunoru/RandomNumbers.jl/blob/838e69e8199941b2331dd98836df669b75b49264/deps/Random123/build.jl#L13

**Comment by sunoru** [comment]
Thanks! I'm really sorry for missing that.

**Comment by ChrisRackauckas** [comment]
Fixed on v1.0.1. Can be closed now.
